SETUP.md

```diff
@@ -0,0 +1,81 @@
+5. Install _Docker Engine_, _containerd_, and the _Docker Compose_ plugin.
+   1. Install Docker's package repository.
+      1. `sudo dnf -y install dnf-plugins-core`
+      2. `sudo dnf config-manager --add-repo https://download.docker.com/linux/fedora/docker-ce.repo`
+   2. `sudo dnf install docker-ce docker-ce-cli containerd.io docker-compose-plugin`
+   3. Start _Docker_: `sudo systemctl start docker`
+   4. Verify that _Docker_ is running: `sudo docker run hello-world`
+   5. Do the post install for _Docker_, (link)[https://docs.docker.com/engine/install/linux-postinstall/]
+      1. Enable the services to start at machine start.
+         - `sudo systemctl enable docker.service`
+         - `sudo systemctl enable containerd.service`
+      2. Allow yourself to execute _Docker_ without elevated privileges.
+         1. `sudo groupadd docker`
+         2. `sudo usermod -aG docker $USER`
+      3. Restart your machine.
+6. Install _Docker Desktop_, [link](https://docs.docker.com/desktop/linux/install/fedora/).
+   1. Download the _RPM_ package, [link](https://docs.docker.com/desktop/release-notes/).
+   2. Install the package: `sudo dnf install /path/to/docker-desktop-<version>-<arch>.rpm`
+7. Enable _Kubernetes_ on _Docker_.
+   1. Start _Docker Desktop_.
+   2. Click on the _Gear_ icon (i.e., settings).
+   3. Click on the _Kubernetes_ menu item.
+   4. Check the _Enable Kubernetes_ checkbox.
+   5. Click the _Apply & Restart_ button.
+8. Register for a _Docker Hub_ account and activate your account, [link](https://hub.docker.com/).
+9. Login to _Docker_, [link](https://docs.docker.com/desktop/linux/#credentials-management).
+   1. `gpg --generate-key`
+   2. `pass init <key>`
+   3. `docker login -u <username>`
+   4. Install _docker-credential-pass_, [link](https://github.com/docker/docker-credential-helpers/releases).
+      1. Download the appropriate tarball for your machine and extract it.
+      2. `sudo cp /path/to/docker-credential-pass /bin/docker-credential-desktop`
+10. Install _kubectl_, [link](https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/).
+    1. `curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"`
+    2. `sudo install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl`
+    3. Test the installation: `kubectl version --client`
+    4. Ensure that _kubectl_ is pointing to the Docker Desktop context:
+       1. `kubectl config get-contexts`
+       2. `kubectl config use-context docker-desktop`
+11. Generate a _Github Personal Access Token_.
+    1. Log into [Github](https://github.com/).
+    2. Go to your account's settings.
+    3. Click on _Developer Settings_ on the left navigation bar.
+    4. Click on _Personal Access Token_.
+    5. Click on the _Generate new token_ button.
+    6. Give the token a descriptive name (e.g., _Constellation_) and no expiration date. Select all scopes except for the delete ones.
```

Comment:
That's wrong and may be insecure. Token should has least-required scopes - currently it's required to fetch the github package only (one of dependencies of the repository). Also, it's wise to set the expiration date and rotate tokens. I'd avoid writing down such things in the official repo.

Reply:
These steps are already documented externally: https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token  Include a link to this instead of repeating.

Also the only scope that is required should be 
>    read:packages
>  Download packages from GitHub Package Registry

I believe, should confirm but that's what I have set.

Reply:
Replaced instructions with the provided link.